panel/browser-panel.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <dlfcn.h>
```

Comment:
#ifndef _WIN32
#include <dlfcn.h>
#endif
Because does not compile on Windows

Reply:
Done. The Linux PR has a different solution for this, we can revert the changes to this file entirely if that lands.